effekt/shared/src/main/scala/effekt/util/DocumentationGenerator.scala

```diff
@@ -0,0 +1,286 @@
+package effekt.util
+
+import effekt.source.*
+import kiama.util.Source
+import scala.collection.immutable.HashMap
+
+// TODO: include spans
+trait DocumentationGenerator {
+  type Documentation = HashMap[String, DocValue]
+  type EffektTree[T <: Tree] = kiama.relation.Tree[AnyRef & Product, T]
```

Comment:
We really wanted to get rid of this dependency :(

Reply:
Yep, already removed it. I just forgot to remove the type definition